src/static/riot/competitions/detail/_tabs.tag

```diff
@@ -246,47 +173,23 @@
                 self.competition.is_admin = CODALAB.state.user.has_competition_admin_privileges(competition)
                 self.update()
                 $('.tabular.menu .item').tab();
+                _.forEach(competition.pages, (page, index) => {
+                    $(`#page_${index}`)[0].innerHTML = sanitize_HTML(EasyMDE.prototype.markdown(page.content))
```

Comment:
Does this ever have race conditions where it tries to set something that doesn't exist?

Reply:
Not that I've seen, but I suppose it could? Are you thinking like trying to set `inner_html` before the `self.update()` has actually created the `<div>`s?

Reply:
> Are you thinking like trying to set inner_html before the self.update() has actually created the <div>s?

Something like that, yeah, if you haven't ever experienced it then it's probably not a huge deal